ambari-web/app/controllers/wizard/step9_controller.js

```diff
@@ -141,7 +141,17 @@ App.WizardStep9Controller = App.WizardStepController.extend(App.ReloadPopupMixin
    * Computed property to determine if the Retry button should be made visible on the page.
    * @type {bool}
    */
-  showRetry: Em.computed.equal('content.cluster.status', 'INSTALL FAILED'),
+  showRetry: function () {
+    const status = this.get('content.cluster.status');
+    switch (status) {
+      case 'INSTALL FAILED':
+      case 'INSTALLED':
+      case 'START FAILED':
+        return true;  
+    }
+
+    return false;
+  }.property('content.cluster.status'),
```

Comment:
currently BE is not correctly handling the state transition from "INSTALLED" to "STARTED", "START_FAILED" or "START_SKIPPED" so this is a temporary workaround. Once that is fixed this will need to be reverted. @mradha25 will open a jira for that.